yaml_edit/lib/src/editor.dart

```diff
@@ -120,15 +120,12 @@ class YamlEditor {
 
   factory YamlEditor(String yaml) => YamlEditor._(yaml);
 
-  YamlEditor._(this._yaml) {
-    ArgumentError.checkNotNull(_yaml);
+  YamlEditor._(this._yaml) : _contents = loadYamlNode(_yaml) {
```

Comment:
@jonasfj I moved the `_contents` instantiation here. Is this what you meant?

Reply:
Love it!